src/chttpd/src/chttpd_changes.erl

```diff
@@ -65,64 +65,85 @@ handle_db_changes(Args0, Req, Db0) ->
     DbName = fabric2_db:name(Db0),
     StartListenerFun = fun() ->
         fabric2_events:link_listener(
-                ?MODULE, handle_db_event, self(), [{dbname, DbName}]
-            )
+            ?MODULE,
+            handle_db_event,
+            self(),
+            [{dbname, DbName}]
+        )
     end,
     Start = fun() ->
-        StartSeq = case Dir =:= rev orelse Since =:= now of
-            true -> fabric2_db:get_update_seq(Db0);
-            false -> Since
-        end,
+        StartSeq =
+            case Dir =:= rev orelse Since =:= now of
+                true -> fabric2_db:get_update_seq(Db0);
```

Comment:
+1 for the ability to put short expressions on the same line as the matching pattern. Some other formaters would always insert new line here.